test/functionalTest/cases/2763_postUpdateContext_error_return/postEntities_with_errors_in_service_path.test

```diff
@@ -53,14 +53,14 @@ echo
 01. Try to create an entity with more than one service path - see error
 =======================================================================
 HTTP/1.1 400 Bad Request
-Content-Length: 79
+Content-Length: 91
 Content-Type: application/json
 Fiware-Correlator: REGEX([0-9a-f\-]{36})
 Date: REGEX(.*)
 
 {
```

Comment:
It seem we were wrongly using NGSIv1-like error response for POST /v2/entity. This PR has raised this and fixed (used to happen during cleanup refactors :)

A bit surprising not to found this in more places. I think it deservers a CNR line like this:

```
- Fix: NGSIv1-like error responses were used in some NGSIv2 operations
```

Reply:
Fixed in edd01ffba0453591e991b12d5f62687890108b09